### ISSUE TITLE
Timings' graph is broken

### ISSUE THREAD

**Issue by Rasgnarok** [SpongePowered/Sponge#1833]
**I am currently running**

- SpongeForge version: SpongeForge 2931
- SpongeVanilla version: N/A
- Java version: Java 8
- Operating System: CentOS 7

- Plugins/Mods: 
![image](https://user-images.githubusercontent.com/18584380/37432188-e6df3766-27ad-11e8-94bf-fd5259ea4312.png)

**Issue Description**
1) Use Nucleus' `/gc` to screenshot the chunk count: 
![image](https://user-images.githubusercontent.com/18584380/37432212-fe68d194-27ad-11e8-82ba-2d44f76e20c2.png)
2) Use Sponge's `/sponge timings paste` to check the chunk count:
https://timings.aikar.co/?id=c457c03d20c44b1f9714223a0ecb0d11
> After a while, results will start looking like this:
https://timings.aikar.co/?id=53af536eb0cf488b857a754a496b9d35
![image](https://user-images.githubusercontent.com/18584380/37432321-5aaf2e30-27ae-11e8-8b8b-6ebcbe8f01e5.png)

tl;dr, Sponge's timings never set to match the chunk count displayed by other plugins, leading to uncertainty regarding how many chunks are actually loaded. Especially important in trying to diagnose whether chunks remain loaded. 
* As an uneducated guess, I believe that timings' never really 'let go' of chunks, or it is displaying how many chunks are still stuck in world save? All I can say is that 20 hours in, my world saves are _really_ long, and I can't tell if it is because of the chunk count, or something else.

**Comment by Rasgnarok** [comment]
Still happens on 3203 Spongeforge.

**Comment by Rasgnarok** [comment]
Another matter regarding timings, that I am unsure if it is related - 
![image](https://user-images.githubusercontent.com/18584380/41615733-09fa55ba-73ca-11e8-80ef-77edd6a1a323.png)

vs


![image](https://user-images.githubusercontent.com/18584380/41615785-2b1957f0-73ca-11e8-806c-99cd1d001f2e.png)


In this case, the first screenshot shows 15TPS with 20% loss, and the second shows 5TPS with 8% loss. Is this supposed to be the case, or is it another bug related to timings' calculations?

**Comment by bloodmc** [comment]
This was actually a bug with the timings website itself and has since been resolved.

**Comment by ryantheleach** [comment]
For reference:

https://github.com/aikar/timings/commit/dc50bb2449d1f74d9997abbc2528b8777d5651db
